src/components/Contentful/Grid.js

```diff
@@ -134,8 +137,8 @@ function GridComponentArrayRenderer({ alignItems, content, id, style }) {
     <GridContainer alignItems={alignItems} id={id} style={style}>
       {content.map((content, i) => (
         <GridCol
-          columnWidth={content.columnWidth}
-          columnOffset={content.columnOffset}
+          columnWidth={content.columnWidth || 12}
```

Comment:
Can we put this default somewhere higher up the chain? Or at least into a config somewhere.